src/resolvers.ts

```diff
@@ -21,8 +21,8 @@ type LoginInfo = {
 
 export const resolvers = {
   Query: {
-    user: async (_, { id }: { id: number }, ctx: UserID) => {
-      if (ctx === null || ctx.id === undefined) {
+    user: async (_, { id }: { id: number }, ctx: GraphQLContext) => {
+      if (!ctx || !ctx.id) {
```

Comment:
o nullable consegue fazer a checagem se o `ctx` existe
```suggestion
      if (!ctx?.id) {
```